test/snapshots.test.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
```

Comment:
I can test every event handler but the "onClickAnywhere", since it installs an event handler to the document. I don't know how to test that one, any idea? 😄